### PR TITLE
PPTP-630: store business registered address

### PR DESCRIPTION
**Pull request by tombart**

### CODE REVIEW

**Review by tmc-ee — APPROVED** [review]

File: app/uk/gov/hmrc/plasticpackagingtaxregistration/models/RegistrationRequest.scala [hunk 1]
```diff
@@ -28,6 +29,7 @@ case class RegistrationRequest(
                  incorpJourneyId = this.incorpJourneyId,
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/plasticpackagingtaxregistration/models/RegistrationRequest.scala:29]: Not relevant to this PR, but this method both could be reduced to `this.copy(id = providerId)` as this is a case class

**Review by SteveSugden — APPROVED** [review]
LGTM

File: app/uk/gov/hmrc/plasticpackagingtaxregistration/models/RegistrationRequest.scala (hunk 1 above)
Comment (tombart) [reply app/uk/gov/hmrc/plasticpackagingtaxregistration/models/RegistrationRequest.scala:29]: just had a look at it, and those are two different classes there: RegistrationRequest and Registration, so this is converting from request to registration.

Comment (SteveSugden) [reply app/uk/gov/hmrc/plasticpackagingtaxregistration/models/RegistrationRequest.scala:29]: Opps.  I was confused by the same field names.  Sorry!

**Pull request merged**